DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>It's difficult to see the median lines -- consider using different colors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Good point, fixed.</p>